susemanager-utils/susemanager-sls/src/modules/mgr_caasp_manager.py

```diff
@@ -0,0 +1,347 @@
+# -*- coding: utf-8 -*-
```

Comment:
@ereslibre : this is the core of what we would be using to call `skuba` via Salt.
Would you please review it if it makes sense from the `skuba` standpoint?